sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -372,6 +397,9 @@ struct _pi_queue : _pi_object {
   ze_fence_handle_t ZeOpenCommandListFence = {nullptr};
   pi_uint32 ZeOpenCommandListSize = {0};
 
+  // Kernels in the open command list.
+  std::vector<pi_kernel> Kernels;
```

Comment:
How is this used?

Reply:
Let's call it something more specific, e.g. OpenCommadListKernels

Reply:
Added comment and more specific name.